src/platform/utilities/feature-toggles/featureFlagNames.js

```diff
@@ -2,6 +2,8 @@
 
 export default Object.freeze({
   bcasLettersUseLighthouse: 'bcas_letters_use_lighthouse',
+  benefitsDocumentsUseLighthouse: 'benefits_documents_use_lighthouse',
+  caregiverSigiEnabled: 'caregiver_sigi_enabled',
```

Comment:
is this `caregiverSigiEnabled` an add by you? having trouble parsing the github diff

Reply:
No, that wasn't added by me, perhaps it was merged in, I'm not sure why it's showing as coming from this branch.